config/serverless/templates/cli-extension.yaml

```diff
@@ -20,6 +20,66 @@ data:
     group: serverless.kyma-project.io
   coreCommands: |
     - actionID: function_init
+      config:
+        python312:
+          depsFilename: requirements.txt
+          depsData: ""
+          handlerFilename: handler.py
+          handlerData: |
+            def main(event, context):
+              message = "Hello World from the Kyma Function "+context['function-name']+" running on "+context['runtime']+ "!";
```

Comment:
```suggestion
              message = "Hello World from the Kyma Function %s running on %s!" % (context['function-name'], context['runtime']);
```

Reply:
It's copied from our docs and busola extension config so if this change makes no functional difference I will stay with the old version to keep examples synchronized

Reply:
ok